COMPONENT_MATURITY.md

```diff
@@ -0,0 +1,170 @@
+
+### Color
+Meets AA color contrast standards for accessibility and color blindness.
+
+#### About
+Code is verified for color contrast using the Axe Core plugin.
+
+Color blindness is tested using a color blindness simulator and visually inspected for:
+- Elements with low color contrast
+- Text readability
+- Link visibility
+- Errors and warnings that might rely on color
+
+#### When to define as 'Not applicable'?
+Should not be used.
+
+#### How to test?
+- Code - Run `yarn test:a11y`
+- Design - Designs are verified using a plugin within Sketch to check color contrast and color blindness.
+
+### Screen readers
+VoiceOver, NVDA, and JAWS were used to test for concise communication and interaction.
+
+#### About
+Components are tested in the following screen readers: 
+- Mac OS
+-- Safari + VoiceOver
+- Windows
+-- NVDA + Edge
+-- JAWS + Edge
+-- iOS + Safari + Mobile VoiceOver (as time permits)
+
+#### When to define as 'Not applicable'?
+When the component has no interactive elements like a button or text input a 'Not applicable' state can be applied.
+
+#### How to test?
+Testing is a manual process and takes some time to understand how screen readers work. Use the instructions provided in the [Accessibility Champions guidance (Google docs)](https://docs.google.com/document/d/1-2W9Gm97lBWWv8ljjjQ20JJEHS4DF8ldqHBIY4yWhFs/edit#bookmark=id.649z7zqdm5au "Accessibility Champions guide") to learn how to use a screen reader. Deque University also provides some guidance and demos on [how to use a screen reader (external link)](https://dequeuniversity.com/screenreaders/ "How to use a screen reader").
+
+### Keyboard navigation
+Component is fully navigable with a keyboard.
+
+#### About
+Keyboard navigation is tested using this [keyboard acceptance criteria (Confluence document)](https://confluence.cms.gov/display/CMSWDS/Keyboard+Navigation+Testing "Keyboard acceptance criteria").
+
+#### When to define as 'Not applicable'?
+When the component has no interactive elements like a button or text input a 'Not applicable' state can be applied.
+
+#### How to test?
+Use this [guidance from WebAIM (external link)](https://webaim.org/techniques/keyboard/ "WebAIM keyboard navigation guidance") to navigate components using a keyboard.
+
+## Code
+
+### Storybook
+Component has stories to cover all defined props.
+
+#### About
+[Storybook (external link)](https://storybook.js.org/ "Storybook JS") is an open-source tool for building components in isolation that is public to all Design System users. 
+
+#### When to define as 'Not applicable'?
+Should not be used.
+
+#### How to test?
+Manually interact with the Storybook stories and make sure that all defined props are in some way tested by our storybook story and that the meaningful ones can be changed. Specifically,
+- Event handlers have visible output through Actions or logging
+- Visual and content props can be changed
```

Comment:
I'm working on improving coverage in this area on our stories, but I would like to qualify this

```suggestion
- Visual and content props can be changed (for props that can be easily changed with standard Storybook controls)
```